protelis.parser.tests/src/org/protelis/parser/tests/ProtelisParsingTest.xtend

```diff
@@ -138,4 +147,30 @@ class ProtelisParsingTest {
 		.shouldNotBeValid
 	}
 
+	@Test
+	def void testIfWithoutParentheses() {
+		'''
+		let x = if (1 < 3) { 1 };
```

Comment:
I think you also need a couple more cases that should parse:
````
let x = if (1<3) { 1 } else { 2 };
1
````
````
let foo = true;
if (foo) { 1 }
2;
````

Reply:
The second is not valid due to the position of the semicolon. Converted to:
```
let foo = true;
if (foo) { 1 };
2
```

Reply:
Right, thank you for fixing that typo.